### PR TITLE
elasticsearch 7.6.1

### PR DESCRIPTION
**Pull request by hirik**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
     <groupId>org.wikimedia.search.highlighter</groupId>
     <artifactId>experimental</artifactId>
-    <version>7.5.2-SNAPSHOT</version>
+    <version>7.6.1-SNAPSHOT</version>
```
Comment (nomoa) [inline pom.xml]: this version needs to be updated in all submodule as well

Comment (hirik) [reply pom.xml]: @nomoa  thanks for the review comments.

**Comment by nomoa** [comment]
Thanks,

there are still compilation issues:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project experimental-highlighter-elasticsearch-plugin: Compilation failure
[ERROR] /srv/nomoa_dev/es/highlighter/experimental-highlighter-elasticsearch-plugin/src/main/java/org/wikimedia/highlighter/experimental/elasticsearch/FieldWrapper.java:[75,71] cannot find symbol
[ERROR]   symbol:   method fullName(java.lang.String)
[ERROR]   location: class org.elasticsearch.index.mapper.MapperService
```

Please make sure the project builds by running: `./mvnw clean verify`

Thanks!

**Comment by hirik** [comment]
@nomoa, i made the changes.

**Pull request closed without merging**